src/client/user/actions/index.ts

```diff
@@ -248,9 +248,14 @@ const closeStatusBar: () => CloseStatusBarAction = () => ({
 const setStatusBarErrorMessage: (
   header: string,
   body: string,
-) => SetStatusBarErrorMessageAction = (header: string, body: string) => ({
+  callbacks: string[],
+) => SetStatusBarErrorMessageAction = (
+  header: string,
+  body: string,
+  callbacks: string[],
+) => ({
   type: UserAction.SET_STATUS_BAR_ERROR_MESSAGE,
-  payload: { header, body },
+  payload: { header, body, callbacks },
 })
```

Comment:
will error messages need to have `callbacks` for the job item URLs? if i'm understanding correctly, it seems strange to allow users to access their S3 links if the latest job failed as a whole

Reply:
For now I'm using it to just ensure that the localStorage message is unique, i.e. if a user has closed a failure message before, and a new failure message appears, the localStorage message needs to be unique, otherwise the message will not re-appear. I could also do it by clearing the localStorage every time a new job is created, but I thought that this was a neater solution because we can probably eventually reuse the callbacks field to retry the failed jobs.

Reply:
AH i didn't think of that at all, sounds neat!

Reply:
Added it into the PR description!